### ISSUE TITLE
no button add implementation in vs code

### ISSUE THREAD

**Issue by Bloody-cpp** [clangd/clangd#1994]
Previously, when I right-clicked on a function or method, I could make a definition in the header file, because of clangd I can no longer do this

Editor/LSP plugin: vs code

Operating system: windows 11

**Comment by HighCommander4** [comment]
When you say "previously", do you mean using a different plugin?

Can you give a code example?

**Comment by Bloody-cpp** [comment]
> When you say "previously", do you mean using a different plugin?
> 
> Can you give a code example?

no, I meant that before I did not use the clangd extension in vs code and could create definitions

example is any function:
void printHelloWorld() noexcept;

**Comment by HighCommander4** [comment]
> Previously, when I right-clicked on a function or method, I could make a definition in the header file

What context menu item did you click on to do this?

> > When you say "previously", do you mean using a different plugin?
> 
> no

Can you show a screenshot of your Extensions panel in VSCode?

**Comment by Bloody-cpp** [comment]
> > Previously, when I right-clicked on a function or method, I could make a definition in the header file
> 
> What context menu item did you click on to do this?
> 
> > > When you say "previously", do you mean using a different plugin?
> > 
> > no
> 
> Can you show a screenshot of your Extensions panel in VSCode?

Now I can’t take a screenshot because it’s night. I just right click on a method or function and at the bottom there was something like "go to definition"
